ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js

```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheckCircle, faArchive, faBolt, faChevronLeft, faPlusCircle } from '@fortawesome/free-solid-svg-icons'
+import { getIconForToken } from '../../../utils/utils'
+import { CustomButton, AddNewTokenNetworkAddress } from '../../../components'
+import rifActions from '../../../actions'
+import { cryptos } from '../../../constants'
+
+class DomainsDetailActiveScreen extends Component {
+	static propTypes = {
+		addNewNetwork: PropTypes.func.isRequired,
+		setAutoRenew: PropTypes.func.isRequired,
+		domainName: PropTypes.string.isRequired,
+		address: PropTypes.string.isRequired,
+		content: PropTypes.string.isRequired, 
+		expirationDate: PropTypes.string.isRequired,
+		autoRenew: PropTypes.bool.isRequired,
+		ownerAddress: PropTypes.string.isRequired,
+		isOwner: PropTypes.bool,
+		isLuminoNode: PropTypes.bool,
+		isRifStorage: PropTypes.bool,
+	}
+	constructor(props) {
+		super(props)
+		let resolvers = []
+		let networks = []
+		Object.assign(resolvers, props.resolvers)
+		Object.keys(cryptos).forEach(function(key) {
+			let crypto = cryptos[key]
+			let network = {
+				value: key,
+				label: crypto.name,
+				icon: crypto.icon,
+				color: crypto.color,
+			}
+			networks.push(network)
+		});
+		this.state = { 
+			resolvers: resolvers,
+			selectedResolverIndex: 0,
+			networks: networks,
+			selectedNetwork: networks[0].value,
+			insertedAddress: '',
+		}
+	}
+	_updateNetwork = (selectedOption) => {
+		this.setState({ selectedNetwork: selectedOption.value })
+	}
+	_updateAddress = (address) => {
+		this.setState({ insertedAddress: address })
+	}
+	_addAddress = () => {
+		let domains = JSON.parse(localStorage.rnsDomains)
+		let myIndex = -1
```

Comment:
I dont like the naming used here. What represents `myIndex` ? Lets use a more descriptive name

Reply:
This variable was renamed, now its called networkIndex